### ISSUE TITLE
Enhancement: stratum.Client should support client.* calls to client 

### ISSUE THREAD

**Issue by jla415** [pocesar/node-stratum#18]
example from server to stratum.Client:

{"id": 1289739, "method": "client.get_version", "params": null}.

Currently the library just throws an exception here:

https://github.com/pocesar/node-stratum/blob/c4decc5be6c66173eac7a5ce76de576c9e9a184c/src/client.ts#L151

see:
https://en.bitcoin.it/wiki/Stratum_mining_protocol#Methods_.28server_to_client.29

**Comment by pocesar** [comment]
yes, I'm going to cover the new protocol changes, the initial version was using slushpool's in 2013
